provision/aws/rds/aurora_create_peering_connection.sh

```diff
@@ -6,13 +6,13 @@ if [[ "$RUNNER_DEBUG" == "1" ]]; then
 fi
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROSA_CLUSTER=$(rosa describe cluster -c ${CLUSTER_NAME} -o json)
```

Comment:
It seems `rosa` command does not work if `AWS_PAGER` is set to `""` which is done in `aurora_common.sh`.

Reply:
I tried the following locally and it worked fine, but I see no harm making the change to be on the safe side:

```
export AWS_PAGER=""
rosa describe cluster -c gh-keycloak -o json
```